ui/app/serializers/pki/certificate.js

```diff
@@ -0,0 +1,3 @@
+import CertSerializer from './cert';
+
+export default class PkiCertificateSerializer extends CertSerializer {}
```

Comment:
Hm, we should probably rewrite this instead of extending but that's just a note for the future

Reply:
Yeah, I think that was the original intention